Grok.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace dokuwiki\plugin\xref;
+
+use dokuwiki\HTTP\DokuHTTPClient;
+
+class Grok
+{
+
+    protected $baseUrl;
+    protected $def;
+    protected $path;
+
+    public function __construct($reference, $baseUrl = 'https://codesearch.dokuwiki.org')
```

Comment:
Url hard code and as default setting available.

Reply:
It's just the default. It is overwritten by the actual config variable when the class is instantiated.